tests/integration/microprofile/config/webapp/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move this to the microprofile/config module

Reply:
it was previously discussed (25.4.2019) and the decision was to extract that from ext/microprofile/config to tests. In ext/microprofile/config we shall have only dependency to Eclipse's microprofile API. And all direct implementations (small rye, helidon) shall be put into tests.

Reply:
Good point